tests/Backend/Workspaces/WorkspacesSnowflakeTest.php

```diff
@@ -836,4 +836,235 @@ public function dataTypesDiffDefinitions()
+                ],
+            ],
+        ];
+        $workspaces->loadWorkspaceData($workspace['id'], $options);
+        $tableRef = $backend->getTableReflection('languages');
+        self::assertEquals(['id', 'name', '_timestamp'], $tableRef->getColumnsNames());
+        self::assertCount(5, $backend->fetchAll('languages'));
+
+        // test workspace load incremental to view
+        $options = [
+            'input' => [
+                [
+                    'source' => $tableId,
+                    'destination' => 'languages',
+                    'incremental' => true,
+                    'useView' => false,
+                ],
+            ],
+            'preserve' => true,
+        ];
+        try {
+            $workspaces->loadWorkspaceData($workspace['id'], $options);
+            self::fail('Incremental load to view cannot work.');
+        } catch (ClientException $e) {
+            // this is expected edge case, view has also _timestamp col
+            // which is ignored when validation incremental load
+            self::assertStringStartsWith('Some columns are missing in source table', $e->getMessage());
+        }
+
+        // do incremental load from file to source table
+        $this->_client->writeTableAsync(
+            $tableId,
+            new CsvFile($importFile),
+            ['incremental' => true]
+        );
+        // test view is still working
+        $tableRef = $backend->getTableReflection('languages');
+        self::assertEquals(['id', 'name', '_timestamp'], $tableRef->getColumnsNames());
+        self::assertCount(10, $backend->fetchAll('languages'));
+
+        // load data from workspace to table
+        $workspace2 = $workspaces->createWorkspace([], true);
+        $options = [
+            'input' => [
+                [
+                    'source' => $tableId,
+                    'destination' => 'languages',
+                ],
+            ],
+        ];
+        $workspaces->loadWorkspaceData($workspace2['id'], $options);
+        $this->_client->writeTableAsyncDirect(
+            $tableId,
+            [
+                'dataWorkspaceId' => $workspace2['id'],
+                'dataObject' => 'languages',
+            ]
+        );
+        // test view is still working
+        $tableRef = $backend->getTableReflection('languages');
+        self::assertEquals(['id', 'name', '_timestamp'], $tableRef->getColumnsNames());
+        self::assertCount(10, $backend->fetchAll('languages'));
+
+//        $fileId = $this->workspaceSapiClient->uploadFile(
+//            (new CsvFile($importFile))->getPathname(),
+//            (new FileUploadOptions())
+//                ->setNotify(false)
+//                ->setIsPublic(false)
+//                ->setCompress(true)
+//                ->setTags(['test-file-1'])
+//        );
+//        // load data from file workspace Not supported yet on S3 on ABS and not used in SNFLK
+//        $fileWorkspace = $workspaces->createWorkspace(
```

Comment:
tady to sem nedělal musel bych to omezit na ABS a nechtěLo sa mě do toho drbat, asi to aj funguje, ale nechal bych to na potom.

Reply:
Nebo `if (Něco::EDGE_CASES_TESTS_ENABLED)` abysme to pak nakonci mohli všechno zapnout a nemuseli hledat kde všude máme něco zacommentované?

Reply:
přidal sem